src/dash.ts

```diff
@@ -1,62 +1,62 @@
+      // Why is this needed? I have no idea, but I think Qt has to start up a listener on cmd before it will work.
+      'win32': 'start dash-plugin:// && start'
+    }[this.OS] || 'zeal';
+  }
+
+  /**
+   * Get command to open dash
+   *
+   * @param {string} query - text to find
+   * @param {string} docsets - array of docset e.g. [css, less]
+   * @return {string} dash handler and uri
+   */
+  getCommand(query: string, docsets: Array<string> = []): string {
+
+    var uri = 'dash-plugin://query=' + encodeURIComponent(query);
+    var keys = this.getKeys(docsets);
+
+    if (keys) {
+      uri += '&keys=' + keys;
     }
 
-    /**
-     * Get command to open dash
-     *
-     * @param {string} query - text to find
-     * @param {string} docsets - array of docset e.g. [css, less]
-     * @return {string} dash handler and uri
-     */
-    getCommand(query: string, docsets: Array<string> = []): string {
+    return this.URIHandler + ' ' + this.getOSSpecificURI(uri);
+  }
 
-        var uri = 'dash-plugin://query=' + encodeURIComponent(query);
-        var keys = this.getKeys(docsets);
+  /**
+   * Get docset keys parameter for dash
+   *
+   * @param {Array<string>} docsets - array of docset e.g [css, less]
+   * @return {string} joined array in string e.g. "ruby,css,dimas" or empty string
+   */
+  getKeys(docsets: Array<string>): string {
 
-        if (keys) {
-            uri += '&keys=' + keys;
-        }
-
-        return this.URIHandler + ' ' + this.getOSSpecificURI(uri);
-    }
-
-    /**
-     * Get docset keys parameter for dash
-     *
-     * @param {Array<string>} docsets - array of docset e.g [css, less]
-     * @return {string} joined array in string e.g. "ruby,css,dimas" or empty string
-     */
-    getKeys(docsets: Array<string>): string {
-
-        if (docsets.length > 0) {
-            return docsets.join(',');
-        }
-
-        return '';
+    if (docsets.length > 0) {
+      return docsets.join(',');
     }
 
-    /**
-     * Returns the OS specific URI to be handled.
-     *
-     * @param {string} uri - original constructed URI
-     * @return {string} OS-specific URI to pass to handler, mainly because of Windows
-     */
-    getOSSpecificURI(uri: string): string {
-        return {
-            // on Windows, start can't deal with the double quotes, and & needs to be escaped with ^
-            'win32': uri.replace('&', '^&')
-        }[this.OS] || '"' + uri + '"';
-    }
+    return '';
+  }
+
+  /**
+   * Returns the OS specific URI to be handled.
+   *
+   * @param {string} uri - original constructed URI
+   * @return {string} OS-specific URI to pass to handler, mainly because of Windows
+   */
+  getOSSpecificURI(uri: string): string {
+    return {
+      // on Windows, start can't deal with the double quotes, and & needs to be escaped with ^
+      'win32': uri.replace('&', '^&')
+    }[this.OS] || '"' + uri + '"';
+  }
```

Comment:
Thank you for making it as two spaces. I just made a new commit to change indentation to 2. After you rebase,  this PR should be cleaner.